docs/pages/management/operations/db-ca-rotation.mdx

```diff
@@ -0,0 +1,225 @@
+---
+title: Database Cert Authority Rotation
+description: How to rotate Teleport database certificate authorities
+---
+
```

Comment:
I would add a quick paragraph about what this page is about and when a user would want to follow it.

Reply:
737e67438f19e3c54de75b0af4013750cbd71344

Reply:
let me know what you think of that as the intro